### DISCUSSION TITLE
FNIRSI GC-01: Cannot flash

### DISCUSSION THREAD

**Discussion by zoloba** [Gissio/radpro#101] [Q&A] [answered]
When I tried to flash Rad Pro 2.0, I saw a startup screen that said 2.0, but it froze at that point. Then the screen went dark and afterwards I could not turn the unit back on. I depressed the reset switch with a pin and was then able to flash the unit's original FNIRSI 1.5 firmware successfully. I'm not sure what I am doing wrong.

![IMG_1332](https://github.com/Gissio/radpro/assets/172933879/d6a65372-ec5a-42e1-997c-0270fb6b0796)

**Comment by Gissio** [comment, marked as answer]
Please read https://github.com/Gissio/radpro/issues/73#issuecomment-2099095107

**Reply by zoloba** [reply]
Thanks. I have ordered a new battery.

**Comment by gc01noob** [comment]
to flash the GC01:

    use Linux
    open the BOOTLOADER drive
    drag over the new firmware .bin
    leave the window open
    drag over the firmware .bin for the second time
    OK the overwrite.
    viola it boots with the new firmware and you can disconnect.
